code/modules/pda/radio.dm

```diff
@@ -1,9 +1,9 @@
-/obj/item/radio/integrated
+/obj/item/integrated_radio

```

Comment:
I reserve a special place in hell for this object.

Someone, way back in the day, device to make `/obj/item/radio/integrated`, and make it distinctly different from `/obj/item/device/radio` The worst part? `/obj/item/radio` doesn't even exist.

This is some abstract item type that should eventually just be refactor into being a *proper* radio, but that's beyond the scope of this PR, and there's enough differences that correcting them would be no easy feat in a short time frame.